Movies.xcodeproj/project.xcworkspace/xcshareddata/swiftpm/Package.resolved

```diff
@@ -5,8 +5,8 @@
       "kind" : "remoteSourceControl",
       "location" : "https://github.com/onevcat/Kingfisher.git",
       "state" : {
-        "revision" : "022e4eeb79f817748544b318b991d9a70036bbf8",
-        "version" : "7.2.4"
```

Comment:
Just a FYI here, the package version is set to `7.0.0 - Next Major` which means it will try to update to latest version until next major (8.0.0). Usually only major versions will contain a breaking change when upgrading, and the key word here is "usually" 😁
If you want to be on the safest side possible it's good to lock the version to exact one number, that way you are sure to only upgrade when you are really meaning to upgrade. Downside is that you loose smaller upgrades and improvements because you now need to do that manually.